sources/T-Editing-Guidelines-201602.adoc

```diff
@@ -0,0 +1,843 @@
+<.<| Text of the Recommendation <.<| 6 onwards {blank}footnote:[When empty, the clause is maintained with an appropriate indication, e.g., "This clause is intentionally left blank.", or "None.".]
+<.<| Annexes (form an integral part of the Recommendation) <.<| A onwards
+<.<| Appendices (do not form an integral part of the Recommendation) <.<| I onwards
+<.<| Electronic attachment <.<| – {blank}footnote:[An electronic attachment may correspond to a clause, an annex, an appendix or may be a non-numbered item.]
+<.<| Bibliography <.<| None
+<.<| Index (optional and rarely used) <.<| None
+|===
+
+[[a_3_1_2]]
+==== annex <<b-ITU-T_A.1>> 
+An annex to a Recommendation contains material (e.g., technical detail or explanation) which is necessary to its overall completeness and comprehensibility and is therefore considered an integral part of the Recommendation. As an integral part of the Recommendation, approval of an annex follows the same approval procedures as Recommendations.
+
+[[a_3_1_3]]
+==== appendix <<b-ITU-T_A.1>> 
+An appendix to a Recommendation contains material which is supplementary to and associated with the subject matter of the Recommendation but is not essential to its completeness or comprehensibility. It is therefore not considered to be an integral part of the Recommendation and thus does not require the same approval procedures as Recommendations; agreement by the study group is sufficient.
+
+[[a_3_1_4]]
+==== clause <<b-ITU-T_A.1>> 
+The word clause shall be used to denote single-digit or multiple-digit numbered text passages.
+
+[[a_3_1_5]]
+==== corrigendum <<b-ITU-T_A.1>>
+A corrigendum to a Recommendation contains corrections to an already published ITU-T Recommendation. A corrigendum is published by ITU-T as a separate document that contains only corrections. TSB may correct obvious errors by issuing a corrigendum with the concurrence of the study group chairman; otherwise, approval of a corrigendum follows the same approval procedures as Recommendations.
+
+[[a_3_1_6]]
+==== implementers' guide <<b-ITU-T_A.1>> 
+An implementers' guide is a document which records all identified defects (e.g., typographical errors, editorial errors, ambiguities, omissions or inconsistencies and technical errors) associated with a Recommendation or a set of Recommendations and their status of correction, from their identification to final resolution. An implementers' guide is issued by ITU-T following agreement by a study group, or following agreement by a working party with concurrence of the study group chairman. Typically, defect corrections are first collected in an implementers' guide and, at a time deemed appropriate by the study group, they are used to produce a corrigendum or are included as revisions to a Recommendation.
+
+[[a_3_2Terms_defined_in_this_Recommendation]]
```

Comment:
@anermina we should rename all the anchors to something that non-machines can understand...

Reply:
I didn't understand the request well, can you please give an example?
b-ITU-T_A.1 is the name of the reference shown in the original paper, while a_3_2Terms_defined_in_this_Recommendation follows the principle from G.650 and Y.3507 - a_sectionNumber sectionTitle

Reply:
@anermina I think instead of `a_3_2Terms_defined_in_this_Recommendation`, the anchor could be just `terms`.

Reply:
Note that the anchors were mechanically assigned by reverse_adoc, based on titles. It is not a necessity to simply those anchors to more brief anchors (such as someone would use if they were writing the document from scratch); but since these are exemplars of ITU markup, I guess that's a harmless request.

@ronaldtse, your request here for short anchors would indeed need to be applied to all the ITU sample docs; and since Manuel is currently working on templates, that would end up being a task for @anermina.

Reply:
@anermina I meant something like this. Notice the reference is actually a `[.source]` element. It is only rendered next to the term in presentation. In semantic markup we do not want to mix concepts.

```adoc

[[terms-implementers-guide]]
==== implementers' guide
An implementers' guide is a document which records all identified defects (e.g., typographical errors, editorial errors, ambiguities, omissions or inconsistencies and technical ...

[.source]
<<b-ITU-T_A.1>> 
```

Reply:
I've fixed this in 142b85d.

Reply:
Sorry, I didn't manage to finish this earlier. I will fix the rest of the cross-references in this document (such as <<a_7Description_of_introductory_material_to_be_provided_by_the_author>>), if it is ok. I've already started doing the same for G.650 document and can do it for the rest of the ITU-T documents as well.

Reply:
@anermina that's great, please proceed.

Reply:
Please remember to run `git pull` to fetch the latest master. Thanks.

Reply:
I've made a new pull request for this document: #41